app/models/container_image.rb

```diff
@@ -36,6 +36,7 @@ class ContainerImage < ApplicationRecord
   after_create :raise_creation_event
 
   def full_name
+    docker_id if image_ref && image_ref.start_with?(DOCKER_PULLABLE_PREFIX)
```

Comment:
@blomquisg :open_mouth: kudos!
@enoodle please fix this here and on master as well.